ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/Sink.java

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package com.mozilla.telemetry.ingestion;
+
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import com.mozilla.telemetry.ingestion.io.Gcs;
+import com.mozilla.telemetry.ingestion.io.Pubsub;
+import com.mozilla.telemetry.ingestion.transform.FileGroupByKey;
+import com.mozilla.telemetry.ingestion.transform.RouteMessage;
+import com.mozilla.telemetry.ingestion.util.Env;
+import com.mozilla.telemetry.ingestion.util.Json;
+import com.mozilla.telemetry.ingestion.util.KV;
+import java.util.concurrent.CompletableFuture;
+
+public class Sink {
+
+  public static void main(String[] args) {
+    // output messages to OUTPUT_BUCKET
+    Gcs.Write gcsWrite = new Gcs.Write(Env.getString("OUTPUT_BUCKET"));
+
+    // output gcs object paths to OUTPUT_TOPIC
+    Pubsub.Write pubsubWrite = new Pubsub.Write(Env.getString("OUTPUT_TOPIC"),
+        Env.getInteger("EXECUTOR_THREADS", 1));
+
+    // write newline delimited strings to files by key
+    FileGroupByKey<String, String> output = new FileGroupByKey<>(
+        Env.getInteger("OUTPUT_BATCH_MAX_BYTES", (int) 1e11), // default 100 GB
+        Env.getDuration("OUTPUT_BATCH_MAX_DELAY", "PT10m"), // default 10 min
+        Env.getInteger("OUTPUT_BATCH_MAX_MESSAGES", (int) 1e9), // default 1bn messages
```

Comment:
I think I'd prefer that the Env class handle accessing the env var names and providing defaults, so that here we're making calls like `Env.outputBatchMaxBytes()` and perhaps rename the class to `Config`.